packages/account/src/Sections/Profile/PhoneVerification/ConfirmPhoneNumber.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PhoneVerificationCard from './PhoneVerificationCard';
+import { Button, Text } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import { Input } from '@deriv/components';
+import { useSettings } from '@deriv/api';
```

Comment:
@suisin-deriv We cant use deri-api in ur legacy accounts package. It wont work with mobx stores